staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -309,7 +309,7 @@ func (wc *watchChan) sync() error {
 		// send items from the response until no more results
 		for i, kv := range getResp.Kvs {
 			lastKey = kv.Key
-			wc.sendEvent(parseKV(kv))
+			wc.queueEvent(parseKV(kv))
```

Comment:
can we drop the method rename to minimize diff for backporting? doing that to improve clarity on master for 1.34 in a follow-up would be fine

Reply:
This was need for merging the logic (that was effectively sendEvent - for consistency with sendError).
But given we're pushing that for follow-up, I agree with it.
Done.